datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/BasicFunctionalityTest.java

```diff
@@ -240,15 +243,19 @@ private void validatePrivateStructAndEncryption(AbsoluteLocation<ResolvedResourc
         // validate encryption on high-level:
         assertThat(foundResource.toString()).doesNotContain(PRIVATE_FILE);
         assertThat(foundResource.toString()).doesNotContain(FOLDER);
-        assertThat(storage.read(foundResource)).asString().doesNotContain(MESSAGE_ONE);
+        try (java.io.InputStream read = storage.read(foundResource)) {
+            assertThat(read).asString().doesNotContain(MESSAGE_ONE);
+        }
     }
 
     @SneakyThrows
     private List<AbsoluteLocation<ResolvedResource>> listFiles(Predicate<String> pattern) {
-        return storage.list(new AbsoluteLocation<>(BasePrivateResource.forPrivate(location)))
-                .filter(it -> !it.location().toASCIIString().startsWith("."))
-                .filter(it -> pattern.test(it.location().toASCIIString()))
-                .collect(Collectors.toList());
+        try (Stream<AbsoluteLocation<ResolvedResource>> stream = storage.list(new AbsoluteLocation<>(BasePrivateResource.forPrivate(location)))) {
```

Comment:
Maybe `ls` instead of `stream` is short and understandable for unix users variable name